qemu/tests/balloon_check.py

```diff
@@ -127,11 +128,15 @@ def _memory_stats_compare(self, keyname, memory_stat_qmp):
         memory_stat_qmp = "%sB" % memory_stat_qmp
         memory_stat_qmp = int(float(utils_misc.normalize_data_size(
                                    memory_stat_qmp, order_magnitude="M")))
-        if (float(abs(guest_mem - memory_stat_qmp)) / guest_mem) > check_mem_ratio:
+        mem_diff = float(abs(guest_mem - memory_stat_qmp))
+        if ((mem_diff / guest_mem) > check_mem_ratio or 
+            mem_diff > check_mem_diff):
```

Comment:
Not sure if I understand it correctly, if you want to tolerant more, should be `and` instead of `or` here`?

Reply:
@vivianQizhu  I think the purpose of this patch is don't make test fail, when only check `check_mem_ratio`.   So `add` is ok.  Only if neither of these two conditions are met, test fail.

Reply:
@yanan-fu @vivianQizhu Thanks for your review.  After a second thought,  I think @vivianQizhu is right. Here we want to avoid failure when guest_mem is small leading absolute difference small.  The purpose is adding another check to make sure the comparison is appropriate. Have upated, thanks!

Reply:
@huangyum  @vivianQizhu  ah~ I misread the code before, what i mean is to use `add` too ...